acceptance/acceptance_test.go

```diff
@@ -3,6 +3,7 @@ package acceptance_test
 import (
 	"bufio"
 	"context"
+	"encoding/base32"
```

Comment:
all changes in acceptance_test.go will disappear after linked PRs are merged and this one is rebased.